python/lsst/pipe/tasks/computeExposureSummaryStats.py

```diff
@@ -69,6 +69,24 @@ class ComputeExposureSummaryStatsConfig(pexConfig.Config):
         dtype=str,
         default="base_SdssShape_psf"
     )
+    psfSampling = pexConfig.Field(
+        dtype=int,
+        doc="Factor by which to subsample the masked image in each dimension for the maxDistToNearestPsf "
```

Comment:
"Factor by which to subsample" reads to me as the number of points in the grid in each dimension, not the step size.  How about "Sampling rate in pixels in each dimension for the maxDistToNearestPsf grid"?